README.md

```diff
@@ -1,37 +1,173 @@
+   ```bash
+   npm run dev
+   ```
+   
+   The site will be available at [http://localhost:5173](http://localhost:5173)
+
+### Available Scripts
+
+| Command                           | Description                              |
+| --------------------------------- | ---------------------------------------- |
+| `npm run dev`                     | Start development server with hot reload |
+| `npm run build`                   | Build the site for production            |
+| `npm run preview`                 | Preview the production build locally     |
+| `npm run generate-og-images`      | Generate Open Graph images               |
+| `npm run generate-contrib-data`   | Generate contributor data                |
+| `npm run generate-hubspot-data`   | Generate HubSpot integration data        |
+| `npm run generate-funding-schema` | Generate funding schema                  |
+
+## Project Structure
+
+```
+spyder-website/
+├── src/
+│   ├── routes/                 # Page routes and content
+│   │   ├── +page.svelte       # Homepage
+│   │   ├── about/             # About page
+│   │   ├── blog/              # Blog posts and pages
+│   │   └── download/          # Download page
+│   ├── lib/                   # Reusable components and utilities
+│   │   ├── blocks/            # Page section components
+│   │   ├── components/        # UI components
+│   │   ├── config/            # Configuration and data
+│   │   ├── i18n/              # Internationalization
+│   │   ├── store/             # State management
+│   │   └── utils/             # Utility functions
+│   ├── app.css                # Global styles
+│   └── app.html               # HTML template
+├── static/                    # Static assets
+│   └── assets/                # Images, fonts, media
+├── scripts/                   # Build and utility scripts
+└── ci/                        # CI/CD scripts
 ```
 
-You can also test the built website with
+## Deployment
+
+This website is automatically deployed to GitHub Pages using GitHub Actions. The deployment process:
+
+1. **Triggers** on push to `main` branch
+2. **Builds** the static site using SvelteKit
+3. **Generates** dynamic content (OG images, contributor data, etc.)
+4. **Deploys** to GitHub Pages
+
+### Manual Deployment
+
+For manual deployment or testing:
 
 ```bash
+# Build the site
+npm run build
+
+# Preview the built site
 npm run preview
 ```
 
-And preview your built site at [https://localhost:5174](https://localhost:5174)
+The built site will be available at [http://localhost:5174](http://localhost:5174)
+
+## Contributing
+
+We welcome contributions from the community! Please see our [Contributing Guide](CONTRIBUTING.md) for details on:
+
+- Setting up the development environment
+- Creating blog posts
+- Making code contributions
+- Submitting pull requests
+- Reporting issues
+
+### Quick Contribution Steps
+
+1. Fork this repository
+2. Create a feature branch (`git checkout -b feature/amazing-feature`)
```

Comment:
```suggestion
2. Create a topic branch (`git switch -c topic-branch-name-here`)
```

- Like many I've moved to calling these topic branches, since they may contain docs, blog posts, infra changes, bug fixes, improvements, etc. not just features
- Don't specify a particular format for branch names that we don't actually use
- Modernize Git syntax